packages/ra-core/src/form/FilterLiveForm.stories.tsx

```diff
@@ -1,14 +1,20 @@
 import * as React from 'react';
 
-import { FilterLiveForm, FilterLiveFormProps, useInput, required } from '.';
+import {
+    FilterLiveForm,
+    FilterLiveFormProps,
+    useInput,
+    required,
+    InputProps,
+} from '.';
 import { ListContextProvider } from '../controller/list/ListContextProvider';
 import { useList } from '../controller/list/useList';
 import { useListContext } from '../controller/list/useListContext';
 
 export default { title: 'ra-core/form/FilterLiveForm' };
 
-const TextInput = props => {
-    const { field, fieldState } = useInput(props);
+const TextInput = ({ defaultValue = '', ...props }: InputProps) => {
+    const { field, fieldState } = useInput({ defaultValue, ...props });
```

Comment:
why do you need to provide a `defaultValue` to `useInput`? I'm worried that `defaultValue = ''` is not the same as `defaultValue = undefined`

Reply:
That avoid a warning in the story about uncontrolled/controlled inputs.